perfzero/lib/cloud_manager.py

```diff
@@ -43,27 +43,19 @@ def run_command(cmd, is_from_user=False):
   Raises:
     Exception: raised when the command execution has non-zero exit code
   """
-  if is_from_user:
-    logging.info('Executing command: %s\n', cmd)
-  else:
-    logging.debug('Executing command: %s\n', cmd)
-
-  stdout = ''
-  p = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                       stderr=subprocess.STDOUT, shell=True)
-  exit_code = None
-  while exit_code is None:
-    exit_code = p.poll()
-    line = p.stdout.readline().decode('utf-8').strip()
-    if not line:
-      continue
-
+  def _log(text):
```

Comment:
It may be slightly simpler to replace this 4-line method with this line:

`_log = logging.info if is_from_user else logging.debug`

Reply:
Thanks, done.